### PR TITLE
Update 30_differential_abundance.qmd

### PR DESCRIPTION
**Pull request by jkc9886**
I updated this code last time but there was a clash with another PR, I incorporated those changes in mine plus the other grammatical changes.

1. the function, subsetbyPrevalent in the original code didn't work when I tried building locally. I replaced it with subsetbyPrevalentFeatures and it is working now, please cross check it.

### CODE REVIEW

File: inst/pages/30_differential_abundance.qmd [hunk 1]
```diff
@@ -1,96 +1,63 @@
...
 ```{r setup, echo=FALSE, results="asis"}
 library(rebook)
 chapterPreamble()
 ```
 
-This section provides an introduction to Differential Abundance Analysis (DAA),
-which is used to identify differences in the abundances of individual taxa (at
-any taxonomic level) between two or more groups, such as treatment and control.
-Here, we demonstrate its implementation on [Tengeler2020](https://microbiome.github.io/mia/reference/Tengeler2020.html) data set.
+Differential Abundance Analysis (DAA) is a method used to identify differences in the abundances of individual taxa (at any taxonomic level) between two or more groups, like treatment versus control groups. In this section, we demonstrate how to implement DAA using the Tengeler2020 data set.
```
Comment (antagomir) [inline inst/pages/30_differential_abundance.qmd:9]: Keep the URLs (it seems to have been removed here)

**Review by antagomir — CHANGES_REQUESTED** [review]

**Comment by antagomir** [comment]
Thanks!

Main issues with this PR:

1) Line breaks have been changed, and as a result it is difficult to compare what exactly has changed because entire text blocks are replaced by other text blocks and sometimes the contents seems identical and sometimes not in a closer investigation -> just to keep the PR more readable, avoid changing line breaks. This does not affect the actual published layout anyway.

2) URLs have been removed in some places.

3) Large chunks of explanations with references seem to have been deleted, not immediately clear if they were just moved to other places in the document or entirely removed.


-> Make a new PR with more modest changes to start with (you can propose the same changes but in a more targeted way, like one subsection at a time to keep the PR more readable)

File: inst/pages/30_differential_abundance.qmd [hunk 2]
```diff
@@ -135,7 +100,7 @@ extensive introduction to its functionality is available in the
 
 ALDEx2 estimates technical variation within each sample per taxon by utilizing
 the Dirichlet distribution. It furthermore applies the CLR transformation (or
-closely related log-ratio transforms). Depending on the experimental setup, it
+                                                                           closely related log-ratio transforms). Depending on the experimental setup, it
```
Comment (TuomasBorman) [inline inst/pages/30_differential_abundance.qmd:103]: Something weird happening

**Comment by TuomasBorman** [comment]
subsetByPrevalent is the new name in mia 1.13.21 https://microbiome.github.io/mia/reference/getPrevalence.html

**Comment by antagomir** [comment]
How this is related to #508

**Comment by jkc9886** [comment]
Do you mean this PR or the discussion about the subsetbyPrevalent function?

**Comment by antagomir** [comment]
There are two open PRs #508 and #514 (this one) and both of them have the same title. If they deal with different parts of the chapter then it is OK. Otherwise there is a risk of conflicts between the two PRs. Can we merge at least one of them soon.

**Comment by jkc9886** [comment]
Hello, yes we can merge this one; the only confusion was with the subsetbyPrevalent which is resolved now. Hence this PR can be merged. Could you do it @TuomasBorman ?

**Comment by TuomasBorman** [comment]
Hello, I forgot, but can you first resolve the issues mentioned i this commit https://github.com/microbiome/OMA/commit/207952f25fc7283c0adb3e6141fad28865a874a6#comments

The book build is failing currently because of previous commits. That must be solved first

**Comment by jkc9886** [comment]
Hello, this commit should be closed and not merged. The PR in which I made the correct charges has already been merged earlier. Sorry for the inconvenience due to the confusion. Also PR#508 can also be closed.

**Pull request closed without merging**